runtime/wasm/src/mapping.rs

```diff
@@ -8,9 +8,18 @@ use std::collections::BTreeMap;
 use std::sync::Arc;
 use std::thread;
 
+const ONE_MIB: usize = 1 << 20; // 1_048_576
+
 lazy_static! {
     /// Verbose logging of mapping inputs
     pub static ref LOG_TRIGGER_DATA: bool = std::env::var("GRAPH_LOG_TRIGGER_DATA").is_ok();
+
+    /// Maximum stack size for the WASM runtime
+    pub static ref MAX_STACK_SIZE: usize = std::env::var("GRAPH_RUNTIME_MAX_STACK_SIZE")
+        .ok()
+        .and_then(|max_stack_size| max_stack_size.parse().ok())
+        // 512KiB
+        .unwrap_or((ONE_MIB as f32 * 0.5) as usize);
```

Comment:
What's wrong with division by 2?

Reply:
Nothing wrong, for some reason I felt like `0.5` was easier to associate with `512KiB`.

Also I think it's easier to change if we want something like `800KiB`.

Reply:
Not that it matters here, but integer arithmetic is much easier to reason about than float arithmetic, if we need to change this to something else we can introduce new constants.

Reply:
Fixed!